### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by hendridg**
In this Milestone 1:

- Configurations Set up, to Linter
- Make a header with logo and menu, and headline with the personal presentation, markup HTML, and Styles

Captura de pantalla de 2021-11-24 23-17-13

### CODE REVIEW

**Comment by carlos-ssh** [comment]
# 🚫⚠️🛑 INVALID SUBMISSION 🚫⚠️🛑
Hi @hendridg , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is missing your linters. Go to your project requirements, check general requirements and create the setup files.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by hendridg** [comment]
Yes, @carlos-ssh, I rename the folder, my mistake, change .githug to .github

**Comment by hendridg** [comment]
Hi Carlos,

Yes, I miss up the name folder .githug, type mistake, I already changed it
to .github

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
Hi @hendridg, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README file (you could use [Microverse's template](https://github.com/microverseinc/readme-template)) and has no linters checks, as you can see here:

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding! 👩‍💻 

keep trying

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by hendridg** [comment]
Done.

**Comment by hendridg** [comment]
I don't know what happens, just delete and put again the Linters file, that's all

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        </h2>
+        <h2>
+            <span>I’m a software developer</span>
+        </h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded,
+            don’t hestiate to contact me.
+        </p>
+        <div class="social-media">
```
Comment (lily-coder) [inline index.html]: - [ ] [REQUIRED] I think the list of icons uses a ul tag to group them, and li per icon; for each icon that has a link (a tag). This is used to present list of information in well formed and semantic way.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <h3>
+                Welcome
+            </h3>
+        </a>
+        <nav>
+            <img src="./assest/Union.svg" alt="menu link">
+        </nav>
+    </header>
+    <section class="headline">
+        <h2>
```
Comment (lily-coder) [inline index.html]: - [ ] [REQUIRED] I think the main title (e.g., "I'm Marianne ...") is a h1 tag. In your case you have used a h2 tag. Please change that as it is part of a good SEO

**Review by lily-coder — CHANGES_REQUESTED** [review]
Hi @hendridg , 

Good job so far!🎉🎉🎉

To Highlight! 🟢
✔️ The  README file is descriptive
✔️ Good job with the linter checks
✔️ Correct Github flow was used

There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Using Imperative tense in the commit messages. In your case you have used past tense.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elyor-doniyorov — APPROVED** [review]
Hi @hendridg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
Approved-d house


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hendridg** [comment]
Complete Milestone 1

**Pull request merged**